app/build.gradle

```diff
@@ -93,7 +93,7 @@ dependencies {
     compile 'com.github.inorichi:ReactiveNetwork:69092ed'
 
     // Android support library
-    final support_library_version = '24.2.1'
+    final support_library_version = '24.1.1'
```

Comment:
this was an accident
though annotation has not been updated so it does cause problems